src/core/custom-elements/rs-changelog.js

```diff
@@ -0,0 +1,85 @@
+// @ts-check
+/**
+ * Fetches commits between two commitish (tag or commit) - `from` and `to=HEAD`
+ * and shows them as a list of "changes during" those commitish. If a filter
+ * function is provided by the user, it is used to filter the commits that are
+ * to be shown. Otherwise, all commits are shown.
+ *
+ * @typedef {{message: string, hash: string}} Commit
+ */
+import { github } from "../github.js";
+import { hyperHTML } from "../import-maps.js";
+import { showInlineError } from "../utils.js";
+
+export const name = "rs-changelog";
+
+export default class ChangelogElement extends HTMLElement {
```

Comment:
Bit weird to use `default` when we don't really do `import element from "./rs-changelog.js"`. How about `export const element = class ...`?

Reply:
We're using `import *` and then `.default`. Agreed that naming it `element` is better.